otter/models/cass.py

```diff
@@ -206,7 +206,7 @@ def serialize_json_data(data, ver):
 
 _cql_count_for_tenant = (
     'SELECT COUNT(*) FROM {cf} '
-    'WHERE "tenantId" = :tenantId;')
+    'WHERE "tenantId"=:tenantId {deleting};')
```

Comment:
Now I'm more confused. I originally asked about `get_counts` (quoted below), which is where you're _not_ passing the "deleting=False" part of the query. Why does that need to know about the deleting groups?

> It's not clear to me why get_counts should return information about deleting groups.